libs/components/toast/src/lib/modules/toast/toast.service.ts

```diff
@@ -49,7 +50,8 @@ export class SkyToastService implements OnDestroy {
       this.#removeHostComponent();
     }
 
-    this.#toastStream.complete();
+    SkyToastService.toasts = [];
+    SkyToastService.toastStream.complete();
```

Comment:
Couldn't this be problematic if two different services are spun up separately? Overwriting the static stream would mean anything in an existing stream would be wiped away - and completing the stream when one service is destroyed would cause the other to no longer work.

Reply:
The service is provided in root and the expectation is that there is only one instance.

Reply:
Correct. Both the legacy and the normal service are provided in "root" (different from our other legacy services) so the only time this destroy hook would get called is if the app is destroyed.

Reply:
But with them both being provided in root - couldn't that mean there is a root instance of `SkyToastService` that is being used one place but another `SkyLegacyToastService` that is being provided somewhere else?

Reply:
Resolving as the chances and real use cases for this are very minimal.